### PR TITLE
add information about what is set up after each step

### LINKED ISSUE #69: Add description of what was created and why for each major step in the readme
Add content to the readme describing which resources and components that were provisioned for each step, and why they're important.

### PR DESCRIPTION
**Pull request by TessFerrandez**
This PR closes #69 by listing the resources provisioned after each step

### CODE REVIEW

File: setup.md [hunk 1]
```diff
@@ -202,6 +213,23 @@ Before we provision the AKS clusters, we will provision some common resources th
...
+* Resource group: **rg-magicaks-shared**
+  * Container registry: **acrmagic123**
+  * Firewall: **fwmagic123**
+  * Key vault: **kv-magic123**
+  * Public IP (for firewall): **pip-firewall**
+  * Route table (for firewall and internet): **subnet-route-table**
+  * Virtual network: **vnet-magicaks**
+    * Subnet (for kubernetes): **snet-k8s**
+    * Subnet (for container instances): **snet-aci**
+    * Subnet: **snet-adhoc**
```
Comment (TessFerrandez) [inline setup.md:227]: Note: I don't know what this subnet is used for

File: setup.md [hunk 2]
```diff
@@ -166,6 +166,17 @@ Terraform stores state configuration in Azure Storage.
...
+After this section you should have the following resources:
+
+* a **magicaksadmins** Azure AD group
+* a **magicaks-terraform** Service Principal for provisioning resources
+* a **magicaks-grafana** Service Principal for operating Grafana
+* azure storage for terraform state
+  * Resource group: **rg-terraform-state**
+    * Storage account: **tfstate[random]**
+      * Container: **tfstate**
+* environment variables set - for running the terraform scripts
```
Comment (tompaana) [inline setup.md:178]:
```suggestion
After this section you should have the following resources created:

* Azure AD group called `magicaksadmins`
* Service principal, `magicaks-terraform`, for provisioning resources
* Service principal, `magicaks-grafana`, for operating Grafana
* Azure storage for Terraform state
  * Resource group: `rg-terraform-state`
    * Storage account: `tfstate[random]`
      * Storage container: `tfstate`
* Environment variables set for running the Terraform scripts
```

Comment (tompaana) [reply setup.md:178]: The emphasis (bold style) on the names of the resources does not make sense to me. I see this is the pattern in other changes too. My personal preference would be to as in my suggestion.

File: setup.md [hunk 3]
```diff
@@ -285,6 +338,17 @@ This will set up Flux for admin and non-admin workloads and apply the desired st
...
 
+After this section you should have the following resources: [given the the default git repository names]
+
+* Kubernetes namespace: **flux-admin**
+  * Helm Release: **flux-admin** for repository **k8smanifests**
+* Kubernetes namespace: **flux-workloads**
+  * Help Release: **flux-workloads** for repository **k8sworkloads**
+* Resource group (created in previous step): **rg-mycluster**
+  * Service bus namespace: **aks-mycluster-servicebus**
+* Key vault secrets:
```
Comment (tompaana) [inline setup.md:349]:
```suggestion
* Key Vault secrets:
```

**Review by tompaana — CHANGES_REQUESTED** [review]
I suggest style changes. It is a matter of opinion so if you disagree, reject and I'll approve.

**Comment by Lybecker** [comment]
Would it make sense to have screenshots of the Azure portal showing the resources created?

**Comment by TessFerrandez** [comment]
> Would it make sense to have screenshots of the Azure portal showing the resources created?

I'll add some screen shots...

File: setup.md (hunk 1 above)
Comment (christoferlof) [reply setup.md:227]: @sachinkundu is it safe to remove the adhoc subnet?

**Review by tompaana — APPROVED** [review]

File: setup.md (hunk 1 above)
Comment (sachinkundu) [reply setup.md]: no its used for example to add bastion box

**Pull request merged**